### PR TITLE
Fixed issue where iFrame <script></script> tags were not parsing corr…

### PR DESCRIPTION
**Pull request by berma77**
…ectly when loading in browser. Added vars where missing. Added notes in README.md. Added test.html to test UglifyJS minified version.

### CODE REVIEW

File: hash_unminified.js [hunk 1]
```diff
@@ -1,545 +1,539 @@
...
+                }
+                catch (e) { }
+            }
+        }
+        return ifr;
+    };
+
+    var sendContainer =  function () {
+        var id = 'sovrn_container';
+        var src = 'http://ap.lijit.com/res/sovrn.containertag.min.js?cid=17&aid=245879';
```
Comment (cpabst) [inline hash_unminified.js]: I'm concerned about the purpose of this container tag call. At the very least, it should be https. But why is this for a single publisher?

File: hash_unminified.js [hunk 2]
```diff
@@ -1,545 +1,539 @@
...
+        return cmn(b ^ c ^ d, a, b, x, s, t);
+    }
+
+    function ii(a, b, c, d, x, s, t) {
+        return cmn(c ^ (b | (~d)), a, b, x, s, t);
+    }
+
+    function calcMD5(str) {
+        var x = str2blks_MD5(str);
+        var a = 1732584193;
```
Comment (cpabst) [inline hash_unminified.js]: Sure this shouldn't be 1732584194? :)

File: hash_minified.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+<script>
```
Comment (cpabst) [inline hash_minified.js]: Why is this only in the minified version?

Comment (berma77) [reply hash_minified.js]: Because that's the only version we will deliver to customers.

**Pull request merged**